src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Listeners/QueueListener.cs

```diff
@@ -42,12 +42,17 @@ internal sealed class QueueListener : IListener, ITaskSeriesCommand, INotificati
             ILoggerFactory loggerFactory,
             SharedQueueWatcher sharedWatcher,
             QueuesOptions queueOptions,
-            QueueProcessor queueProcessor = null,
```

Comment:
The QueueProcessor param was only added to support forced overrides for some of our special internal queue listeners. Moving to a simpler model where a factory is always required/used, and the special instances just implement their own factory.